hydromt_fiat/workflows/hazard.py

```diff
@@ -308,71 +327,97 @@ def fstrip(x):
             #             "var": None if not hasattr(self, "var_lst") else self.var_lst[idx],
             #             "chunks": "auto" if chunks == "auto" else self.chunks_lst[idx],
             #            }
-
-            da = da.expand_dims({"rp": [da_rp]}, axis=0)
+            
+            
 
             model_fiat.set_maps(da, da_name)
             # post = f"(rp {da_rp})" if rp is not None and risk_output else ""
             post = f"(rp {da_rp})" if risk_output else ""
             model_fiat.logger.info(f"Added {hazard_type} hazard map: {da_name} {post}")
 
-            # new_da = xr.concat([new_da, da], dim='rp')
+            # new_da = xr.concat([new_da, da], dim='rp')   
```

Comment:
Can you delete the lines from this whole file that are commented out and that we are for sure not using at a later stage?